src/main/java/net/starlark/java/syntax/NodeVisitor.java

```diff
@@ -174,9 +174,7 @@ public void visit(@SuppressWarnings("unused") Comment node) {}
   public void visit(ConditionalExpression node) {
     visit(node.getCondition());
     visit(node.getThenCase());
-    if (node.getElseCase() != null) {
```

Comment:
Why is this change safe?

Reply:
`ConditionalExpression#getElseCase()` is not annotated as `@Nullable`, which is correct since the lexer [only accepts a `ConditionalStatement` if it has an `else`](https://cs.opensource.google/bazel/bazel/+/6201dd161aa2878efc68cfcd6a2acf9193f548e8:src/main/java/net/starlark/java/syntax/Parser.java;l=944;drc=7e3755d3bf1443a0f2a780bc8efff658c464c3a5).